### PR TITLE
OSSMDOC-397: Ability to skip route creation for specific Gateways

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6-4.11

Issue: https://issues.redhat.com/browse/OSSMDOC-397

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-397/service_mesh/v2x/ossm-traffic-manage.html#ossm-auto-route-enable_routing-traffic

Additional information:
Eng review; jwendell
QE review: skondkar
Peer review: jstickler

### CODE REVIEW

**Review by jwendell — APPROVED** [review]

**Comment by skondkar** [comment]
Verified with setting ServiceMeshControlPlane field - gateways.openshiftRoute.enabled to false as per the preview doc. LGTM

File: modules/ossm-auto-route-enable.adoc [hunk 1]
```diff
@@ -7,6 +7,14 @@
 
 By default, the `ServiceMeshControlPlane` resource automatically synchronizes the Gateway resources with OpenShift routes. Disabling the automatic route creation allows you more flexibility to control routes if you have a special case or prefer to control routes manually.
 
+== Skipping automatic route creation for specific cases
```
Comment (JStickler) [inline modules/ossm-auto-route-enable.adoc:10]: "All titles and section headings must have an anchor ID. The anchor ID must be similar to the title or section heading."
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

File: modules/ossm-auto-route-enable.adoc [hunk 2]
```diff
@@ -7,6 +7,14 @@
...
+
+If you want to skip the automatic management of OpenShift routes for a specific Istio Gateway, you must add the annotation `maistra.io/manageRoute: false` to the Gateway metadata definition. {SMProductName} will ignore Istio Gateways with this annotation, while keeping the automatic management of the other Istio Gateways.
```
Comment (JStickler) [inline modules/ossm-auto-route-enable.adoc:12]: Can we change "skip" to "disable" here?

File: modules/ossm-auto-route-enable.adoc [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
+
+== Disabling automatic route creation for all cases
```
Comment (JStickler) [inline modules/ossm-auto-route-enable.adoc:14]: Same as above, needs an anchor ID.

**Review by JStickler — CHANGES_REQUESTED** [review]
Need some IDs for the new headings.

File: modules/ossm-auto-route-enable.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-auto-route-enable.adoc]: Made the change, updated the PR, and generated a new preview build.

File: modules/ossm-auto-route-enable.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-auto-route-enable.adoc]: Made the change, updated the PR, and generated a new preview build.

File: modules/ossm-auto-route-enable.adoc (hunk 3 above)
Comment (rh-tokeefe) [reply modules/ossm-auto-route-enable.adoc:16]: Made the change, updated the PR, and generated a new preview build.

File: modules/ossm-auto-route-enable.adoc [hunk 4]
```diff
@@ -7,6 +7,16 @@
 
 By default, the `ServiceMeshControlPlane` resource automatically synchronizes the Gateway resources with OpenShift routes. Disabling the automatic route creation allows you more flexibility to control routes if you have a special case or prefer to control routes manually.
 
+[id="disabling-automatic-route-creation-specific-cases"]
```
Comment (JStickler) [inline modules/ossm-auto-route-enable.adoc:10]: [id="disabling-automatic-route-creation-specific-cases_{context}"]

Needs to have context at the end of each ID, not just the first one in the file, as shown in the guidelines.
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

Comment (rh-tokeefe) [reply modules/ossm-auto-route-enable.adoc]: @JStickler this has been updated. I looked at anchoring in an assembly by mistake. Thanks!

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**